Source/Scene/GroundPrimitive.js

```diff
@@ -825,10 +943,19 @@ define([
             return;
         }
 
-        if (!defined(GroundPrimitive._maxHeight)) {
+        if (!GroundPrimitive._initialized) {
+            if (!this.asynchronous) {
+                throw new DeveloperError('For synchronous GroundPrimitives, you must call GroundPrimitive.initializeTerrainHeights() and wait for the returned promise to resolve.');
```

Comment:
Should this be a `RuntimeError`? If not, wrap it in pragmas to be remove when built.